streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/entity/Application.java

```diff
@@ -202,10 +203,12 @@ public class Application implements Serializable {
 
   private String description;
 
+  @TableField(fill = FieldFill.INSERT)
   private Date createTime;
 
   private Date optionTime;
 
+  @TableField(update = "now()", fill = FieldFill.INSERT_UPDATE)
   private Date modifyTime;
```

Comment:
It is not advisable to use annotations. This will trigger an update whenever there is a change in the records in the table, which does not meet our expectations. The `modifyTime` is only updated when users manually modify entity info on the frontend. Often, StremaPark will modify records in the table, e.g: When the system detects a change in the job status. that needs to be synchronized to the database. In these cases, we do not want to update the `modifyTime`.